src/main/java/com/zendesk/maxwell/row/RowMap.java

```diff
@@ -364,18 +280,11 @@ public String toJSON(MaxwellOutputConfig outputConfig) throws Exception {
 		g.flush();
 
 		if(outputConfig.encryptionMode == EncryptionMode.ENCRYPT_ALL){
-			String plaintext = jsonFromStream();
-			encryptingJsonGeneratorThreadLocal.get().writeEncryptedObject(plaintext, encryptionContext);
-			g.flush();
+			String plaintext = json.consume();
+			json.getEncryptingGenerator().writeEncryptedObject(plaintext, encryptionContext);
```

Comment:
what about the call to `g.flush` here?  was it superfluous?

Reply:
It's part of `MaxwellJson.consume()`. I found a couple other places where we were still calling it unnecessarily, so I've removed those too.